nameko/cli/shell.py

```diff
@@ -53,11 +83,17 @@ def main(args):
     ctx = {}
     ctx['n'] = make_nameko_helper(config)
 
-    # Support the regular Python interpreter startup script if someone
-    # is using it.
-    startup = os.environ.get('PYTHONSTARTUP')
-    if startup and os.path.isfile(startup):
-        with open(startup, 'r') as f:
-            eval(compile(f.read(), startup, 'exec'), ctx)
+    try:
+        if args.plain:
+            raise ImportError
+
+        run_shell(shell=args.interface, banner=banner, local=ctx)
+    except ImportError:
+        # Support the regular Python interpreter startup script if someone
```

Comment:
from what i can tell (using latest versions on pypi, might differ between versions), calling `embed` in bpython causes pythonstartup to be executed, but in ipython. one option is to run it "manually" for all shells (like this), and then unset the environment variable to prevent it from running again. thoughts?